### PR TITLE
table/tables: fix panic

### PR DESCRIPTION
**Pull request by coocood**
fix panic

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: table/tables/index.go [hunk 1]
```diff
@@ -278,7 +278,7 @@ func (c *index) Exist(rm kv.RetrieverMutator, indexedValues []types.Datum, h int
 func (c *index) FetchValues(r []types.Datum) ([]types.Datum, error) {
 	vals := make([]types.Datum, len(c.idxInfo.Columns))
 	for i, ic := range c.idxInfo.Columns {
-		if ic.Offset < 0 || ic.Offset > len(r) {
+		if ic.Offset < 0 || ic.Offset >= len(r) {
```
Comment (ngaut) [inline table/tables/index.go]: Should we log more details ?

**Comment by zimulala** [comment]
In addIndices function add error check，line421 in tables.go

**Comment by zimulala** [comment]
LGTM @ngaut

**Comment by ngaut** [comment]
LGTM

**Pull request merged**